pages/docs/reference/whatsnew1420.md

```diff
@@ -0,0 +1,348 @@
+specify the option in the `pod {}` build script block of a dependency like this:
+
+```kotlin
+pod("foo") {
+   extraOpts = listOf("-Xforeign-exception-mode”, “objc-wrap")
+}
+```
+
+The default behavior remains unchanged: the program terminates when an exception is thrown from the Objective-C code.
+
+### Support for Xcode 12 libraries
+    
+We have added support for new libraries delivered with Xcode 12. Now you can use them from the Kotlin code.
+
+### Performance improvements and bug fixes
+
+As usual, Kotlin/Native receives performance improvements and bug fixes in various components, including the ones added
+in 1.4.0, for example, the [code sharing mechanism](mpp-share-on-platforms.html#share-code-on-similar-platforms). 
+
+[**Back to top**](#)
+
+## Kotlin Multiplatform
+
+### Updated structure of multiplatform library publications 
+
+Starting from Kotlin 1.4.20, there is no longer a separate metadata publication. Metadata artifacts are now included in
+the _root_ publication which stands for the whole library and is automatically resolved to the appropriate platform-specific
+artifacts when added as a dependency to the common source set.
+
+Learn more about publishing a multiplatform library [here](https://kotlinlang.org/docs/reference/mpp-publish-lib.html).
+
+#### Compatibility with earlier versions
+
+This change of structure breaks the compatibility between projects with [hierarchical project structure](mpp-share-on-platforms.html#share-code-on-similar-platforms).
+If a multiplatform project and a library it depends on both have the hierarchical project structure, then you need to update
+them to Kotlin 1.4.20 or higher simultaneously. Libraries published with Kotlin 1.4.20 are not available for using from
+project published with earlier versions.
+
+Projects and libraries without the hierarchical project structure remain compatible.
+
+[**Back to top**](#)
+
+## CocoaPods support
+
+### Improved task execution
+
+In this release, we significantly improved task execution flow. For example, if you add a new CocoaPods dependency,
+existing dependencies are not rebuilt. Adding an extra target also doesn't affect rebuilding dependencies for existing ones.
+
+### Extended DSL
+
+In 1.4.20, we extend the DSL of adding [CocoaPods](native/cocoapods.html) dependencies to your Kotlin project.
+
+In addition to local Pods and Pods from the CocoaPods repository, you can add dependencies on the following types of libraries:
+* A library from a custom spec repository.
+* A remote library from a Git repository.
+* A library from an archive (also available by arbitrary HTTP address).
+* A static library.
+* A library with custom cinterop options.
+
+Learn more about adding CocoaPods dependencies in Kotlin projects [here](native/cocoapods.html#add-dependencies-on-pod-libraries).
+Find examples in the [Kotlin with CocoaPods sample](https://github.com/Kotlin/kotlin-with-cocoapods-sample).
+
+### Updated integration with Xcode
```

Comment:
I would consider moving it closer to other Xcode improvements.
Cocoapods relate to both – MPP and Native.

We can discuss how to structure better. But it's strange to read about Xcode there and then here after a few sections.